modbuscommander/integrationpluginmodbuscommander.cpp

```diff
@@ -387,7 +387,15 @@ void IntegrationPluginModbusCommander::thingRemoved(Thing *thing)
     qCDebug(dcModbusCommander()) << "Removing thing" << thing->name();
     if (thing->thingClassId() == modbusTCPClientThingClassId) {
         ModbusTCPMaster *modbus = m_modbusTCPMasters.take(thing);
+        if (modbus) {
```

Comment:
how can it happen that in thingRemoved() the modbus object is missing?

Reply:
well, only if you have a bug and you firget to add it...you are right, the check is really not necessary